e2e/terraform/shared/consul/consul_aws.service

```diff
@@ -6,7 +6,7 @@ After=network-online.target
 [Service]
 Restart=on-failure
 Environment=CONSUL_ALLOW_PRIVILEGED_PORTS=true
-ExecStart=/usr/local/bin/consul agent -config-dir="/etc/consul.d" -dns-port="53" -recursor="172.31.0.2"
+ExecStart=/usr/local/bin/consul agent -config-dir="/etc/consul.d" -recursor="172.31.0.2"
```

Comment:
If we've removed `systemd-resolvd` to get rid of the port conflict between it and Consul, why do we need to remove the `-dns-port="53"` and add `dnsmasq`?

Reply:
when I removed resolvd I was getting issues with any sort of dns resolution (mostly failing curls) I followed the https://learn.hashicorp.com/consul/security-networking/forwarding guide which made it seem like 8600 for consul was the way to go but I admit I was playing wack-a-mole at the time.

Reply:
Well if there's a learn doc that describes it, then it's going to be a scenario we see from users so it can't hurt to exercise it here.